dm/dm/master/server.go

```diff
@@ -563,7 +563,7 @@ func (s *Server) StartTask(ctx context.Context, req *pb.StartTaskRequest) (*pb.S
 				return respWithErr(terror.Annotate(err, "while putting task command line arguments"))
 			}
 		}
-		err = s.scheduler.AddSubTasks(latched, subtaskCfgPointersToInstances(stCfgs...)...)
+		err = s.scheduler.AddSubTasks(latched, pb.Stage_Running, subtaskCfgPointersToInstances(stCfgs...)...)
```

Comment:
maybe we can rename this `StartTask`(create + start task in current spec) to difference it with current `start task`, maybe later.